tests/fixtures/metadata/step-summary-plat-ue2-sandbox-test-drift-54.md

```diff
@@ -0,0 +1,394 @@
+                      - Resource  = "arn:aws:s3:::cptest-plat-ue2-sandbox-test-drift-54-01/*"
+                      - Sid       = "DenyUnEncryptedObjectUploads"
+                    },
+                  - {
+                      - Action    = "s3:*"
+                      - Condition = {
+                          - Bool = {
+                              - "aws:SecureTransport" = "false"
+                            }
+                        }
+                      - Effect    = "Deny"
+                      - Principal = "*"
+                      - Resource  = [
+                          - "arn:aws:s3:::cptest-plat-ue2-sandbox-test-drift-54-01/*",
+                          - "arn:aws:s3:::cptest-plat-ue2-sandbox-test-drift-54-01",
+                        ]
+                      - Sid       = "ForceSSLOnlyAccess"
+                    },
+                ]
+              - Version   = "2012-10-17"
+            }
+        ) -> (known after apply)
+    }
+
+  # module.s3_bucket.aws_s3_bucket_public_access_block.default[0] must be replaced
+-/+ resource "aws_s3_bucket_public_access_block" "default" {
+      ~ bucket                  = "cptest-plat-ue2-sandbox-test-drift-54-01" # forces replacement -> (known after apply) # forces replacement
+      ~ id                      = "cptest-plat-ue2-sandbox-test-drift-54-01" -> (known after apply)
+        # (4 unchanged attributes hidden)
+    }
+
+  # module.s3_bucket.aws_s3_bucket_server_side_encryption_configuration.default[0] must be replaced
+-/+ resource "aws_s3_bucket_server_side_encryption_configuration" "default" {
+      ~ bucket = "cptest-plat-ue2-sandbox-test-drift-54-01" # forces replacement -> (known after apply) # forces replacement
+      ~ id     = "cptest-plat-ue2-sandbox-test-drift-54-01" -> (known after apply)
+
+        # (1 unchanged block hidden)
+    }
+
+  # module.s3_bucket.aws_s3_bucket_versioning.default[0] must be replaced
+-/+ resource "aws_s3_bucket_versioning" "default" {
+      ~ bucket = "cptest-plat-ue2-sandbox-test-drift-54-01" # forces replacement -> (known after apply) # forces replacement
+      ~ id     = "cptest-plat-ue2-sandbox-test-drift-54-01" -> (known after apply)
+
+      ~ versioning_configuration {
+          + mfa_delete = (known after apply)
+            # (1 unchanged attribute hidden)
+        }
+    }
+
+Plan: 8 to add, 0 to change, 5 to destroy.
+
+Changes to Outputs:
+  ~ bucket_arn                  = "arn:aws:s3:::cptest-plat-ue2-sandbox-test-drift-54-01" -> (known after apply)
+  ~ bucket_domain_name          = "cptest-plat-ue2-sandbox-test-drift-54-01.s3.amazonaws.com" -> (known after apply)
+  ~ bucket_id                   = "cptest-plat-ue2-sandbox-test-drift-54-01" -> (known after apply)
+  ~ bucket_region               = "us-east-2" -> (known after apply)
+  ~ bucket_regional_domain_name = "cptest-plat-ue2-sandbox-test-drift-54-01.s3.us-east-2.amazonaws.com" -> (known after apply)
+```
+
+</details>
+
+
+
+      
```

Comment:
I feel like we should fix the template, rather than add all this white space to all the tests.